src/Language/PureScript/Docs/RenderedCode/RenderType.hs

```diff
@@ -154,12 +154,6 @@ insertPlaceholders :: RenderTypeOptions -> Type -> Type
 insertPlaceholders opts =
   everywhereOnTypesTopDown convertForAlls . everywhereOnTypes (convert opts)
 
-dePrim :: Type -> Type
-dePrim ty@(TypeConstructor (Qualified _ name))
-  | ty == tyBoolean || ty == tyNumber || ty == tyString =
-    TypeConstructor $ Qualified Nothing name
-dePrim other = other
```

Comment:
Oh just to clarify: `psc-docs` output shouldn't change as a result of this PR either. Also here's the part of Pursuit where this information is used, in case you're interested: https://github.com/purescript/pursuit/blob/681559794a31e3de18452124a718ab09c64aaec6/src/Model/DocsAsHtml.hs#L201-L216